starknet_devnet/blueprints/rpc/transactions.py

```diff
@@ -193,9 +205,9 @@ async def estimate_fee(request: List[RpcBroadcastedTxn], block_id: BlockId) -> l
         )
     except StarkException as ex:
         if "Entry point" in ex.message and "not found" in ex.message:
-            raise RpcError.from_spec_name("INVALID_MESSAGE_SELECTOR") from ex
+            raise RpcError.from_spec_name("CONTRACT_ERROR") from ex
```

Comment:
What motivated this change?

Reply:
These errors were removed from the spec.